modules/doc/content/getting_started/installation/manual_installation_llvm.md

```diff
@@ -60,7 +58,6 @@ export C_INCLUDE_PATH=$PACKAGES_DIR/mpich-__MPICH__/include:$C_INCLUDE_PATH
 export CPLUS_INCLUDE_PATH=$PACKAGES_DIR/mpich-__MPICH__/include:$CPLUS_INCLUDE_PATH
 export FPATH=$PACKAGES_DIR/mpich-__MPICH__/include:$FPATH
 export MANPATH=$PACKAGES_DIR/mpich-__MPICH__/share/man:$MANPATH
-export PETSC_DIR=$PACKAGES_DIR/petsc-__PETSC_DEFAULT__
```

Comment:
Just to be clean, since we're removing PETSC_DEFAULT from the bash profile, we should also remove mention of PETSC in the head section about (on line 51)